### ISSUE TITLE
Serialization Issue

### ISSUE THREAD

**Issue by NielsPilgaard** [TelepathicGrunt/UltraAmplifiedMod#8]
Hi o/

I believe biome features of Ultra Amplified Dimension are serialized incorrectly, which can cause issues when other mods try to generate them - In this case, the mod [TerraForged](https://github.com/TerraForged/TerraForged).

It doesn't cause any issues because UAD biome features aren't used in the Overworld, but I thought you'd prefer to have it reported anyhow.

### Version
- ultra_amplified_dimension-1.15.2-7.2.3

### Log
 https://gist.github.com/NillerMedDild/0874052ed233231120ee0e1f8c99b749

Original report on TerraForged's issue tracker: https://github.com/TerraForged/TerraForged/issues/29

**Comment by TelepathicGrunt** [comment]
Huh. It seems TerraForge tries to serialize feature which can cause a crash as I find many mods does not set up their features to be able to be serialized. I'll see what I can do to make my features be serialize and also let the TerraForge developer know about this so he can add a check in case a different mod also can causes the crash.

**Comment by NielsPilgaard** [comment]
Oh it doesn't crash, it's merely noted in the logs :) The supplied log is just a latest.log full of NPEs.
I already informed the TerraForged devs, but thanks!

**Comment by TelepathicGrunt** [comment]
So I talked with TerraForge dev and it turns out that the serialization thing you saw was just some debugging stuff he was using and that he will be changing it in the next version of his mod.

I did find out that the feature's weren't serializing because some of them weren't registered correctly but now they are and shouldn't cause log spam in other mods that serializes features. 

Issue is now resolved for v7.2.4 of Ultra Amplified and so, this issue will be closed now. Thank you for reporting this!
